todo_app/templates/index.html

```diff
@@ -2,17 +2,29 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
+<form action="/item" method="post" class="todo-form" id="todoForm">
   <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <h1 class="todo-header" id="todoHeader">To-Do App</h1>
+    <p class="todo-header-para" id="todoHeaderPara">Things to-do app.</p>
   </div>
-
+  <!-- <div class="jumbotron"> -->
+    <h2 class="add-item-input add-item-submit" id="addItemInputSubmit">Add Item</h1>
+    <input type="text" name="item-input" value=""></input>
+    <button type="submit">Add item</button>
+  <!-- </div> -->
+  <br>
+  <br>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %} 
+        <li>
+          {{ item.title }} - {{ item.status }}
+        </li> 
+        {% endfor %} 
       </ul>
     </div>
   </div>
+</form>
```

Comment:
Again just a small thing but I'd put this `</form>` up at where the form actually ends (which I think is around line 14